jobs/e2e_node/image-config-swap.yaml

```diff
@@ -0,0 +1,9 @@
+# To copy an image between projects:
+# `gcloud compute --project <to-project> disks create <image name> --image=https://www.googleapis.com/compute/v1/projects/<from-project>/global/images/<image-name>`
+# `gcloud compute --project <to-project> images create <image-name> --source-disk=<image-name>`
+images:
+  ubuntu: #With1GSwap
```

Comment:
Per the [spec](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2400-node-swap/README.md#test-plan), we want to test two distros. Ubuntu is a great start, can we also do something RHEL-based?

Reply:
Done. Added fedora image with 1G swap and swap accounting enabled.